strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/configuration/HttpConnectionPoolConfigurationManagementController.java

```diff
@@ -48,34 +48,24 @@ public HttpConnectionPoolConfigurationManagementController(ConfigurationManageme
     @PutMapping(value = "{storageId}/{repositoryId}/{numberOfConnections}",
                 produces = { MediaType.TEXT_PLAIN_VALUE,
                              MediaType.APPLICATION_JSON_VALUE })
-    public ResponseEntity setNumberOfConnectionsForProxyRepository(@PathVariable(value = "storageId") String storageId,
-                                                                   @PathVariable(value = "repositoryId") String repositoryId,
+    public ResponseEntity setNumberOfConnectionsForProxyRepository(@RepositoryMapping Repository repository,
                                                                    @PathVariable(value = "numberOfConnections") int numberOfConnections,
                                                                    @RequestHeader(HttpHeaders.ACCEPT) String accept)
     {
-        Storage storage = getConfiguration().getStorage(storageId);
-        if (storage == null)
-        {
-            return ResponseEntity.status(HttpStatus.NOT_FOUND)
-                                 .body(getResponseEntityBody("The storage does not exist!", accept));
-        }
-
-        ImmutableRepository repository = (ImmutableRepository) storage.getRepository(repositoryId);
-        if (storage.getRepository(repositoryId) == null)
-        {
-            return ResponseEntity.status(HttpStatus.NOT_FOUND)
-                                 .body(getResponseEntityBody("The repository does not exist!", accept));
-        }
+        final ImmutableRepository immutableRepository = (ImmutableRepository) repository;
+        final String storageId = immutableRepository.getStorage().getId();
+        final String repositoryId = immutableRepository.getId();
 
-        if (((ImmutableRepository)repository).getRemoteRepository() == null)
+        if (immutableRepository.getRemoteRepository() == null)
         {
             return ResponseEntity.status(HttpStatus.BAD_REQUEST)
                                  .body(getResponseEntityBody("The proxy repository has no associated remote repository.", accept));
         }
 
         configurationManagementService.setProxyRepositoryMaxConnections(storageId, repositoryId, numberOfConnections);
-        proxyRepositoryConnectionPoolConfigurationService.setMaxPerRepository(repository.getRemoteRepository().getUrl(),
-                                                                              numberOfConnections);
+        proxyRepositoryConnectionPoolConfigurationService.setMaxPerRepository(
+                immutableRepository.getRemoteRepository().getUrl(),
+                numberOfConnections);
```

Comment:
@carlspring we should finally decide something with common formatting style we all should use.

I prefer how it was before this change.

Reply:
@sbespalov : I've restored the previous formatting.